hooks/src/InputField.tsx

```diff
@@ -0,0 +1,36 @@
+export interface InputFormProps {
+  title: string;
+  label?: string;
+  description?: string;
+  feedbackMessage?: string;
+  children: React.ReactNode;
+}
+
+function InputField(props: InputFormProps) {
+  const { title, label, description, feedbackMessage } = props;
+
+  return (
+    <div className="fieldContainer">
+      <div className="titleBox">
+        <h3 className="title tx-xl">{title}</h3>
+        {description && <p className="tx-md">{description}</p>}
+      </div>
+      <div className="inputBox">
+        <label className="label tx-lg">{label}</label>
+        <div className="inputContainer">{props.children}</div>
+        <p
+          style={
+            feedbackMessage
+              ? { visibility: 'visible' }
+              : { visibility: 'hidden' }
+          }
```

Comment:
추가로 여기는 소소하게 아래처럼 적는게 더 깔끔하지 않을까 싶네요 (취향이 담긴 부분이라 반영 안 해주셔도 됩니다)

```suggestion
          style={{ visbility: feedbackMessage ? 'visible' : 'hidden' }}
```

Reply:
오 훨씬 깔끔하고 좋은 것 같습니다! 👍 의견 감사합니다!

Reply:
인라인 스타일의 삼항 연산자 로직을 가독성 있게 개선했습니다!

[refactor: 인라인 스타일 가독성 개선 (c53cd1c)](https://github.com/woowacourse/react-modules/pull/157/commits/c53cd1ce05745fc15f108ef3d10952f3f61751c6)